src/python/grapl-tests-common/grapl_tests_common/clients/engagement_edge_client.py

```diff
@@ -0,0 +1,34 @@
+import requests
+from typing import Optional
+from http import HTTPStatus
+
+_JSON_CONTENT_TYPE_HEADERS = {"Content-type": "application/json"}
+
+
+class EngagementEdgeException(Exception):
+    pass
+
+
+class EngagementEdgeClient:
+    def __init__(self, use_docker_links: bool = False) -> None:
+        hostname = "grapl-engagement-edge" if use_docker_links else "localhost"
+        self.endpoint = f"http://{hostname}:8900"
+
+    def get_jwt(self) -> str:
+        resp = requests.post(
+            f"{self.endpoint}/login",
+            json={
+                # hardcoded when IS_LOCAL
+                "username": "grapluser",
+                "password": "2ae5ddfb1eeeed45d502bcfd0c7b8f962f24bf85328ba942f32a31c0229c295a",
```

Comment:
Isn't the password graplpassword locally?

Reply:
it's peppered and sha'd
(i briefly toyed with doing the pepper/sha here, but after 10 minutes of the resulting sha not being what i expected, i just decided 'fuck it')